### ISSUE TITLE
Décider d'une identité graphique et créer une maquette de la page d'accueil

### ISSUE THREAD

**Issue by pierreminiggio** [pierreminiggio/touslesverbes.com#1]
Avant de développer le front, il faut se poser la question de ce qu'on veut bien afficher comme texte sur la page d'accueil & les couleurs.
Informations que l'internaute doit savoir quand il va sur la page d'accueil : 
- Qu'on est un site qui liste tous les verbes dans la langue française
- Qu'on en a fait une API, et on mettra un lien vers la documentation.
- Qu'on peut participer au projet, (on mettra un lien vers les 2 repos github)
- Autres ?

**Comment by QuentinLagree** [comment]
On pourrais mettre un exemple de code pour montrer la simplicité du code et faire envie à l'utilisateur, non ?

**Comment by QuentinLagree** [comment]
Sur quel logiciel ou site web on fait la maquette ?

**Comment by pierreminiggio** [comment]
Comme tu préfères pour le logiciel.
Et tu fais référence à quoi comme exemple de code ? Sur comment utiliser l'API ?

**Comment by QuentinLagree** [comment]
En logiciel, je pense que je vais utiliser Figma comme ça on pourra travailler en team. Oui, je fais référence à comme utiliser l’api.

**Comment by pierreminiggio** [comment]
Yup ça peut être une idée pour donner un exemple d'utilisation avec une des routes.
Pour la liste exhaustive des routes, il y a la doc qui le fait : 
https://api.touslesverbes.com/doc

**Comment by QuentinLagree** [comment]
Tu as déjà des idées pour les couleurs ?

**Comment by pierreminiggio** [comment]
Nope j'ai aucune idée de couleur.

**Comment by QuentinLagree** [comment]
Tu pense à un site assez moderne avec des couleurs vives ?

**Comment by QuentinLagree** [comment]
Pour trouver les couleurs j'utilise coolors.fr et donc dis moi ce que tu en pense de cette palette ( pour la couleur principale) : https://coolors.co/09f79d-08f07c-0ee77c-0be374-0ae376

**Comment by pierreminiggio** [comment]
ça me semble bien

**Comment by QuentinLagree** [comment]
Ca marche, comme couleur principale je prends celle là. Et tu as une idée de logo, pour l'identité graphique c'est primordiale.

**Comment by pierreminiggio** [comment]
Nope j'ai 0 idée de logo.

**Comment by pierreminiggio** [comment]
![image](https://user-images.githubusercontent.com/35456349/106366732-4152ce00-633e-11eb-9754-8cc78e3e27f4.png)
Je colle ce que m'a envoyé @QuentinLagree pour que ça soit historisé ici : 
![image](https://user-images.githubusercontent.com/35456349/106366740-529bda80-633e-11eb-8e9b-2a804db0bbb0.png)

**Comment by QuentinLagree** [comment]
Mais du coup tu en pense quoi ?

**Comment by QuentinLagree** [comment]
Alors je suis aussi parti sur du bleu parce que je trouvais que le vert était trop vif et donc pas assez de contraste entre les boutons et le fond. Je te met la maquette sur ce lien Figma : https://www.figma.com/file/JtKOJFolyqBYcMXYNyjT1K/Touslesverbes.com?node-id=0%3A1

**Comment by QuentinLagree** [comment]
Pour la page d'accueil du compte mettre une nav bar ? si oui dit moi les liens que tu voudrais mettre

**Comment by pierreminiggio** [comment]
Les liens importants je pense que ça serait la doc de l'API : https://api.touslesverbes.com
Et pourquoi pas aussi les liens des 2 projets Github : 
https://github.com/pierreminiggio/touslesverbes.com
https://github.com/pierreminiggio/api.touslesverbes.com

**Comment by QuentinLagree** [comment]
Ok ça marche
